packages/node/test/machine/integration/random-signing-keys.ts

```diff
@@ -12,13 +13,22 @@ export function getSortedRandomSigningKeys(length: number) {
     );
 }
 
-export function getRandomHDNodes(length: number) {
-  // tslint:disable-next-line:prefer-array-literal
+export function getXPubsFromExtendedKeys(xkeys: string[]): string[] {
```

Comment:
I'd make `xkeyToXpub` a function and then instead of doing `getXPubsFromExtendedKeys(xkeys)` in the code, just do `xkeys.map(neuterExtendedKey)`